Assets/Scripts/CameraController.cs

```diff
@@ -33,12 +33,11 @@ void LateUpdate()
         Camera.main.transform.Rotate(new Vector3(0,1,0), horizontal * cameraSpeed * dt);
         Camera.main.transform.Rotate(new Vector3(1,0,0), vertical * cameraSpeed * dt);
```

Comment:
Please do no use Camera.main to perform multiple operations as it is a very costly function. Instead use it to get a reference to the main camera in the Start/Awake function and use the reference when doing the operations

Reply:
TIL. Here I assumed it was a cached reference in the Camera class rather than a FindObjectWithTag wrapper